docs/src/install-github-ci.md

```diff
@@ -22,26 +22,30 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: 'Checkout your code.'
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: 'Run Kani on your code.'
-        uses: model-checking/kani@vVER.SION
+        uses: model-checking/kani-github-action@vVER.SION
```

Comment:
Why `vVER.SION`?

Reply:
I would prefer something like: `v<MAJOR>.<MINOR>`. Also, what will happen if they provide a patch number?

Reply:
done.  

For now, we haven't supported patches. I would expect that in the case we needed to file a patch, it would be because of a critical bug, and we would want the CI to move to the patched version.

Reply:
That's about to change... :)